src/components/TrustedSenders.vue

```diff
@@ -24,7 +24,7 @@
 		<div v-for="sender in sortedSenders"
 			:key="sender.email">
 			{{ sender.email }}
-
+			{{ t('mail', '{type}', {type: sender.type}) }}
```

Comment:
this doesn't translate anything. the translation string only contains a placeholder but nothing else to translate.

you can do the following: add a method for `senderType(type)` with a `switch` statement. for individual and domain you can return a translated string. for anything else you fall back to `type`.